src/librustc_typeck/check/method/suggest.rs

```diff
@@ -130,6 +131,15 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
                                    "candidate #{} is defined in the trait `{}`",
                                    idx + 1,
                                    self.tcx.item_path_str(trait_did));
+                        err.span_help(span,
```

Comment:
I would make this a non-span help, since the span is already part of the error, I don't think it is necessary to show the span twice.